### DISCUSSION TITLE
Real time log in simulator is not working anymore with zeus dev command

### DISCUSSION THREAD

**Discussion by idriss21** [zepp-health/zeppos-forum#395] [Developer Feedback]
Hello , 

I was using the simulator to develop an app that works on the Zepp V1 GTS3. Recently, I haven't been able to see the real-time logs (device logs inside page/index.js) when I save changes to the source code using the Zeus dev command. I’ve tried several solutions, like installing Tunnelblick and running the simulator with root permissions, but none of these solutions have worked for me.

Im using MacOS v14.6.1 .

<img width="1048" alt="Screenshot 2024-12-04 at 10 15 44 PM" src="https://github.com/user-attachments/assets/77a88e90-a477-44ad-a214-58f21c7afa13">

Thanks so much for your support ,

**Comment by silver-zepp** [comment]
Hey @idriss21 the simulator v1.x guide is **not applicable** to the sim v2 and rather can worsen the situation with all the sudo strings attached. I'd suggest: 
1. removing the sim
2. removing the `~/.zepp` folder
3. reinstalling the sim + emus

If that still doesn't work, run the sim from the terminal and show us the output

**Reply by idriss21** [reply]
thanks for support after removing .zepp folder and installing a fresh new version things started working
